webdriver/tests/take_element_screenshot/user_prompts.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from tests.support.asserts import assert_png, assert_success
+from tests.support.asserts import assert_png, assert_success, assert_error
```

Comment:
nit: mind listing the imported helper functions in alphabetical order?

Reply:
Fixed.